Core/Controller/EditAgenciaTransporte.php

```diff
@@ -35,7 +37,7 @@ class EditAgenciaTransporte extends ExtendedController\EditController
      */
     public function getModelName()
     {
-        return 'FacturaScripts\Core\Model\AgenciaTransporte';
+        return AgenciaTransporte::class;
```

Comment:
Aquí da igual lo que diga phpstorm. En la beta cambiaremos estos return 'FacturaScripts\Core\Model\AgenciaTransporte' por 'FacturaScripts\Dinamic\Model\AgenciaTransporte';

Reply:
Mejor como estaba.

Reply:
Como te he dicho antes, lo cambio sin problemas, si es cierto que de esa manera dirá que no lo encuentra al pasarlo a Dinamic y al verlo tal cual, era algo que ni había contemplado. (Ya no respondo lo mismo en el resto de comentarios que hacen mención al mismo cambio)

No lo dice PhpStorm, lo dice PHP Code Sniffer, y manda a esta info: https://secure.php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class y el mensaje que da es de este estilo. Sería bueno que todos lo usaramos, porqué en bastantes casos sugiere cambios para mejorar eficiencia y coincide con lo que dice Scrutinizer.

Perhaps this can be replaced with \FacturaScripts\Core\Model\Agente::class. less... (Ctrl+F1) 
The feature documentation can be found here.